src/docfx/docset/Context.cs

```diff
@@ -47,6 +47,15 @@ public void Report(Error error)
             _report.Write(error);
         }
 
+        /// <summary>
+        /// Convert relative path to absolute path
+        /// </summary>
+        /// <param name="relativePath">Relatived to _outputPath</param>
+        public string ConvertToAbsolutePath(string relativePath)
```

Comment:
>ConvertToAbsolutePath [](start = 22, length = 21)

doesn't make sense to put this method to Context, would we put it to Config.Output?

Reply:
or just use Path.Combine in legacy